src/main/java/org/variantsync/diffdetective/diff/difftree/serialize/edgeformat/ChildOrderEdgeFormat.java

```diff
@@ -0,0 +1,12 @@
+package org.variantsync.diffdetective.diff.difftree.serialize.edgeformat;
+
+import org.variantsync.diffdetective.diff.difftree.serialize.StyledEdge;
+
+public class ChildOrderEdgeFormat extends EdgeLabelFormat {
+    @Override
+    public String labelOf(StyledEdge edge) {
+        int i = edge.from().indexOfChild(edge.to());
+        int j = edge.to().indexOfChild(edge.from());
+        return String.format(";%d", i < 0 ? j : i);
+    }
+}
```

Comment:
Please add documentation on author and how labels are formatted.

Reply:
Documentation and `@author` added.